### ISSUE TITLE
999 Error Code

### ISSUE THREAD

**Issue by amirtechs** [ProxymanApp/Proxyman#922]
### Proxyman version? (Ex. Proxyman 1.4.3)
Proxyman 2.28.0 (22800)

### macOS Version? (Ex. mac 10.14)
Mac 11.2.3

### Description
Hi there, previously it's working fine. However, after I enable SSL when debugging my iOS Simulator it's return error. When Proxyman is launched, it blocks the internet access to the browser and iOS Simulator. Had tried reinstall and format the certificate, the issue still persisted.

### Steps to reproduce 
1. Launch proxyman
2. Monitor the logs

### Expected behavior
1. User should be able to view the successful code (eg; 200 or 201) instead of 999

### Screenshots (optional)
<img width="1303" alt="Screenshot 2021-07-05 at 4 20 24 PM" src="https://user-images.githubusercontent.com/25275137/124440576-56300380-ddad-11eb-9193-31dc1ea64cb0.png">

**Comment by NghiaTranUIT** [comment]
When you enable SSL Proxying for Google Chrome. Are you able to access google.com or other websites?

Maybe you're using VPN app? 🤔

If not, please try Factory Reset in Help Menu -> Advanced to see if it works for you 👍

**Comment by NghiaTranUIT** [comment]
For the iOS Simulator, Have you installed and trusted the certificate on your iOS Simulator 🤔 ?

You can do it automatically by using Certificate menu -> Install for iOS -> Simulator and follows these steps

**Comment by cameroncooke** [comment]
I've also seen the 999 error quite a lot which I don't really understand because I'm sure `999` is not a valid HTTP status code. It feels like Proxyman is handling some kind of internal error by returning a response from the proxy which doesn't feel right to me.

**Comment by NghiaTranUIT** [comment]
yes, 999 is an internal code. I suppose that I should remove it, and improve the wording for common SSL error from Swift NIO 👍

**Comment by senolatac** [comment]
I got this error also for Appstore:
Factory Reset didn't work.
<img width="941" alt="Screen Shot 2023-02-17 at 17 25 33" src="https://user-images.githubusercontent.com/27292083/219681356-d6d1b9f4-b0c3-49fc-a840-d6d139d55b15.png">

**Comment by NghiaTranUIT** [comment]
@senolatac you could not intercept AppStore traffic because it's protected by SSL Pinning. It's a security mechanism to protect from Man-in-the-middle apps like Proxyman, Charles Proxy, and Fiddler.

There is no solution to bypass the SSL Pinning unless you hold the Pinned certificate of the AppStore.

**Comment by senolatac** [comment]
@NghiaTranUIT I can listen network with frida + jailbroken-device + objection + proxyman... The problem is initial request.

**Comment by NghiaTranUIT** [comment]
so, you mean, only the initial requests (from AppStore) is error, other requests are still working fine 🤔

**Comment by senolatac** [comment]
yes exactly @NghiaTranUIT

**Comment by AKGhub123** [comment]
When I tried to run Zebra device simulator, the error returned in Proxy man as follows:
![image](https://github.com/user-attachments/assets/a77af9fc-8cbf-49a6-95c5-5576316f3f64)

**Comment by Chevel** [comment]
Issue still happening. Proxyman Version 5.10.0 (51000) | macOS: Version 14.7 (Build 23H124)
Started happening after updating to macos14.7 and xcode16.1
